api/src/main/java/org/pmiops/workbench/utils/mappers/CommonMappers.java

```diff
@@ -126,4 +134,12 @@ public Long getInitialCreditsExpiration(
       DbUser source, @Context InitialCreditsService initialCreditsService) {
     return initialCreditsService.getCreditsExpiration(source).map(this::timestamp).orElse(null);
   }
+
+  @Named("getBillingStatus")
```

Comment:
Shift getBillingStatus into mapper, because it requires access to config which should not (cannot?) be accessed from an entity.